frontend/packages/client/src/components/HomeHeader.tsx

```diff
@@ -10,7 +10,13 @@ const stars = [
   { topPer: 0.92, leftPer: 0.67, width: '25px', height: '25px', start: 500 },
 ];
 
-export default function HomeHeader({ isVisible = true } = {}) {
+interface HomeHeaderProps {
+  isVisible: boolean;
+}
+
+export default function HomeHeader({
+  isVisible = true,
+}: HomeHeaderProps): JSX.Element | null {
```

Comment:
Could we start declaring things like this?: 
```
export const HomeHeader: React.FC<HomeHeaderProps> = ({ isVisible = true }) => {
  return ....;
};
```

FC has null as a return type as well and makes is a bit more readable (IMO), although using JSX.Element we need to be more explicit in the types declaration (stronger types) and that's good too.

Reply:
Yea I'll switch to use React.FC